src/TableView/TableActions.js

```diff
@@ -308,6 +309,16 @@ function deleteError(error) {
   };
 }
 
+function _previousPageIfEmptyAfterDeletion(numberDeleted, state, plural, dispatch) {
```

Comment:
this function is just an encapsulation of the fixing of a bug, since you need it in two places.